src/pybamm/telemetry.py

```diff
@@ -3,12 +3,15 @@
 import pybamm
 import sys
 
+
 _posthog = Posthog(
     # this is the public, write only API key, so it's ok to include it here
     project_api_key="phc_bLZKBW03XjgiRhbWnPsnKPr0iw0z03fA6ZZYjxgW7ej",
     host="https://us.i.posthog.com",
 )
 
+_posthog.log.setLevel("CRITICAL")
```

Comment:
The error from not being connected to the internet comes out at the `ERROR` level. This stops the warning as far as I can tell